### PR TITLE
CPOL-55 Support new RBAC

### PR DESCRIPTION
**Pull request by josejulio**
This is what it currently looks like when no permissions
Screenshot_2020-02-04_14-20-33

~~I'll remove the rbac-client and test with the insights-chrome https://github.com/RedHatInsights/insights-chrome/pull/554 as a followup~~
I'll change when this is ready to test on ci

### CODE REVIEW

**Comment by pilhuhn** [comment]
Can you please change the text in the bulls-eye from "Unable to connect" to something along the lines of "you have no permission"?

**Comment by josejulio** [comment]
> Can you please change the text in the bulls-eye from "Unable to connect" to something along the lines of "you have no permission"?

Yes!
Also I'll keep using the `rbac-client` directly until insights chrome releases its own

**Comment by josejulio** [comment]
> Can you please change the text in the bulls-eye from "Unable to connect" to something along the lines of "you have no permission"?

done

File: src/pages/ListPage/PolicyTableError.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        return {
+            icon: ExclamationCircleIcon,
+            title: 'No permission to view this page',
+            content: 'You do not have permission to view this page'
+        };
+    }
+
+    if (requestHasError) {
+        switch (httpCode) {
+            case 404:
```
Comment (pilhuhn) [inline src/pages/ListPage/PolicyTableError.ts]: I think this should explicitly test for 403 as well (basically the `!canReadAll` case)

**Comment by pilhuhn** [comment]
I think the code looks ok, but I can not get it to work with a user that is valid, but does not have rbac access to custom-policies.
In that case the rbac call still returns *:* , but the calls to the backend return a 403 (as the backend sees that the user has no permissions). The result is that the UI does not disable the create button and also shows "Unable to connect" instead of "No permission"

File: src/pages/ListPage/PolicyTableError.ts (hunk 1 above)
Comment (josejulio) [reply src/pages/ListPage/PolicyTableError.ts]: > I think this should explicitly test for 403 as well (basically the `!canReadAll` case)

done

**Pull request merged**